src/runtime-helpers/custom-functions.ts

```diff
@@ -0,0 +1,4 @@
+interface IInvocationContext<T> {
```

Comment:
I am wondering if we could make the declaration of a streaming custom function simpler by using a simpler name for this type. For example, ICaller<T>.

Reply:
`IStreamingCustomFunction`
or
`IStreamingFunction`